lib/matplotlib/__init__.pyi

```diff
@@ -70,14 +70,22 @@ def get_cachedir() -> str: ...
 def get_data_path() -> str: ...
 def matplotlib_fname() -> str: ...
 
-class RcParams(dict[str, Any]):
+class RcParams(MutableMapping):
     validate: dict[str, Callable]
+    namespaces: tuple
+    single_key_set: set
```

Comment:
Can these be more specific about the type of the values?

e.g. `tuple[str, ...]` (this is how you say homogeneous tuple of str, indeterminate length)

Reply:
Ah, thanks! I wasn't sure how to explain indeterminate length so kept it as just a tuple. I'll update it.